realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -224,9 +229,23 @@ private void emitAccessors(JavaWriter writer) throws IOException {
                 // Getter
                 writer.emitAnnotation("Override");
                 writer.beginMethod(fieldTypeCanonicalName, metadata.getGetter(fieldName), EnumSet.of(Modifier.PUBLIC));
-                writer.emitStatement(
-                        "return new RealmList<%s>(%s.class, row.getLinkList(%s), realm)",
-                        genericType, genericType, staticFieldIndexVarName(field));
+
+                writer.emitSingleLineComment("use the cached value if available");
```

Comment:
Do we add comments to proxy classes? I mean, it is useful if you are going to read a proxy class but we don't expect our users to do so.